packages/react-native-payments/lib/js/NativePayments.js

```diff
@@ -77,7 +77,7 @@ const NativePayments: {
     });
   },
 
-  handleDetailsUpdate(details) {
+  handleDetailsUpdate(details, invalidShippingAddress) {
```

Comment:
@deemaabdallah should we have a default such as `invalidShippingAddress=false`? So there are no breaking changes?